cerbero/packages/wix_packager.py

```diff
@@ -57,17 +57,17 @@ def pack(self, output_dir, devel=False, force=False, keep_temp=False):
         return paths
 
     def create_merge_module(self, output_dir, package_type, force, version,
-                            keep_temp):
+                            keep_temp, keep_strip_temp_dir=False):
```

Comment:
I don't think you need this variable, you only have to keep it when using fragments

Reply:
Yes, but as the method returns different kind of values depending on this variable value (a single string or a duple of strings), I prefer keeping it explicit on method parameters than depending on an internal state.

So you know exactly what kind of data to expect as a return value at the moment of calling the method.